### PR TITLE
Optimizations

### PR DESCRIPTION
**Pull request by akxcv**
- Rewrote `CounterSegment` and `StaticCounterDigits` as functions
- Configured Webpack to build with `NODE_ENV=production` for acceptance testing (on CI)
- Replaced `babel-preset-stage-0` with plugins for object rest spread and class properties

Minified file size shrunk from 20.6 kB to 17.4 kB 👍

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
     "lint-staged": "lint-staged",
     "spellcheck": "yaspeller-ci *.md",
     "test": "yarn run test:unit && yarn run test:acceptance && yarn run lint && yarn run spellcheck",
-    "test:acceptance": "karma start",
-    "test:acceptance:watch": "WATCH=true yarn run test:acceptance -- --no-single-run",
+    "test:acceptance": "NODE_ENV=production karma start",
```
Comment (JelF) [inline package.json]: why don't you just add NODE_ENV=production on CI servers?

File: src/components/CounterSegment.js [hunk 2]
```diff
@@ -27,100 +50,66 @@ const PERIOD_LIMITS = {
  *   digitWrapper={(digit) => digit}
  * />
  */
-class CounterSegment extends React.Component {
+export default function CounterSegment (props) {
```
Comment (JelF) [inline src/components/CounterSegment.js]: Can you describe this transformation or give a link to something describing it?

Comment (akxcv) [inline src/components/CounterSegment.js]: It's just basic React stuff. If you don't need things like constructor or lifecycle methods, your component can be a function that returns whatever `render` of a class component would return. This is significantly more performant.

File: package.json (hunk 1 above)
Comment (akxcv) [inline package.json]: Should I? I think, this way tests will be more consistent on local machines, too.

Comment (JelF) [inline package.json]: @akxcv on local machines, acceptance tests was good to check compicated things during development

**Comment by akxcv** [comment]
@JelF updated.

**Pull request merged**